packages/devextreme-cli/utility/layout.js

```diff
@@ -0,0 +1,31 @@
+const runPrompts = require('../utility/prompts');
+
+const getLayoutFromOptions = (options, layouts) => {
+    if(!options.layout) {
+        return;
+    }
+
+    const currentLayout = layouts.filter((layout) => {
+        const layoutName = layout.fullName || layout.value;
+        return layoutName === options.layout;
+    });
+
+    return currentLayout.length ? [currentLayout[0].value] : undefined;
+};
```

Comment:
```suggestion
const getLayoutFromOptions = ({ layout:layoutName }, layouts) => {
    return layoutName && layouts.find((layout) => (layout.fullName || layout.value) === layoutName)?.value;
};
```
And why we pass whole `options` object to this method if we need only one property?

Reply:
I will fix it